index.html

```diff
@@ -19,101 +20,97 @@ <h1>FizzBuzz問題</h1>
         <td><input type="text" id="buzz" placeholder="整数値を入力してください"></td>
     </tr>
 </table>
-<button type="button" id="btn1">実行</button>
+<button type="button" id="btn">実行</button>
 </form>
 <p>【出力】</p>
 </section>
-<section id="section2">
+<section id="output">
 </section>
 <script>
 //実行
-const ele_btn1 = document.getElementById('btn1');
-ele_btn1.addEventListener('click', (e) => {
+const eleBtn = document.getElementById('btn');
+eleBtn.addEventListener('click', (e) => {
 
     //タグの用意
-    let ele_sec2 = document.getElementById('section2');
-    let ele_nums = document.createElement('dev'); 
+    const eleOut = document.getElementById('output');
+    const eleNums = document.createElement('dev'); 
 
     //初期化
-    if(ele_sec2.hasChildNodes()) {
-        while(ele_sec2.firstChild){
-            ele_sec2.removeChild(ele_sec2.firstChild);
+    if(eleOut.hasChildNodes()) {
+        while(eleOut.firstChild){
+            eleOut.removeChild(eleOut.firstChild);
         }
     }
 
     //テキストフィールドのデータ入力
-    const input_fizz = document.getElementById("fizz").value;
-    const input_buzz = document.getElementById("buzz").value;
+    const inputFizz = document.getElementById('fizz').value;
+    const inputBuzz = document.getElementById('buzz').value;
 
     //空チェック
-    if(!input_fizz || !input_buzz){
-        let ele_err = document.createElement('p');
-        ele_err.textContent = "半角数字を入力してください";
-        ele_nums.appendChild(ele_err);
+    if(!inputFizz || !inputBuzz){
+        const eleErr = document.createElement('p');
+        eleErr.textContent = '半角数字を入力してください';
+        eleNums.appendChild(eleErr);
```

Comment:
エラーチェックの場合は以下のような感じで `return` を書いて
「一番最後の else がエラーがない時」を避けるように記述することが多いです。
```suggestion
        eleNums.appendChild(eleErr);
        return;
```

Reply:
上記修正致しました。
よろしくお願いします